### ISSUE TITLE
[bug]:  Validation error for CheckpointConfig config

### ISSUE THREAD

**Issue by mirek190** [invoke-ai/InvokeAI#4711]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

24

### What version did you experience this issue on?

3.2-rc2

### What happened?

During use the program I wanted to generate next picture ( previously I made around 20-30 images ) and suddenly I couldn't generate the picture ...so I closed the CMD condole and run it again .
Right now I can not run InvokeAI   ( choose 1 )

error:

`````
1. Generate images with the browser-based interface
2. Explore InvokeAI nodes using a command-line interface
3. Run textual inversion training
4. Merge models (diffusers type only)
5. Download and install models
6. Change InvokeAI startup options
7. Re-run the configure script to fix a broken install or to complete a major upgrade
8. Open the developer console
9. Update InvokeAI
10. Run the InvokeAI image database maintenance script
11. Command-line help
Q - Quit
Please enter 1-11, Q: [1] 1
Starting the InvokeAI browser-based UI..
[2023-09-27 00:03:39,016]::[InvokeAI]::INFO --> Patchmatch initialized
E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\torchvision\transforms\functional_tensor.py:5: UserWarning: The torchvision.transforms.functional_tensor module is deprecated in 0.15 and will be **removed in 0.17**. Please don't rely on it. You probably just need to use APIs in torchvision.transforms.functional or in torchvision.transforms.v2.functional.
  warnings.warn(
[2023-09-27 00:03:41,663]::[uvicorn.error]::INFO --> Started server process [20992]
[2023-09-27 00:03:41,663]::[uvicorn.error]::INFO --> Waiting for application startup.
[2023-09-27 00:03:41,664]::[InvokeAI]::INFO --> InvokeAI version 3.2.0rc2
[2023-09-27 00:03:41,664]::[InvokeAI]::INFO --> Root directory = E:\DIFFUSION\InvokeAI
[2023-09-27 00:03:41,664]::[InvokeAI]::INFO --> Using database at E:\DIFFUSION\InvokeAI\databases\invokeai.db
[2023-09-27 00:03:41,665]::[InvokeAI]::INFO --> GPU device = cuda NVIDIA GeForce RTX 3090
[2023-09-27 00:03:41,682]::[InvokeAI]::INFO --> Scanning E:\DIFFUSION\InvokeAI\models for new models
[2023-09-27 00:03:41,694]::[uvicorn.error]::ERROR --> Traceback (most recent call last):
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\starlette\routing.py", line 671, in lifespan
    async with self.lifespan_context(app):
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\starlette\routing.py", line 566, in __aenter__
    await self._router.startup()
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\starlette\routing.py", line 648, in startup
    await handler()
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\app\api_app.py", line 77, in startup_event
    ApiDependencies.initialize(config=app_config, event_handler_id=event_handler_id, logger=logger)
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\app\api\dependencies.py", line 131, in initialize
    model_manager=ModelManagerService(config, logger),
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\app\services\model_manager_service.py", line 341, in __init__
    self.mgr = ModelManager(
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\model_manager.py", line 354, in __init__
    self._read_models(config)
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\model_manager.py", line 377, in _read_models
    self.scan_models_directory()
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\model_manager.py", line 997, in scan_models_directory
    model_config: ModelConfigBase = model_class.probe_config(
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\models\base.py", line 206, in probe_config
    return cls.create_config(
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\models\base.py", line 202, in create_config
    return configs[kwargs["model_format"]](**kwargs)
  File "pydantic\main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for CheckpointConfig
config
  field required (type=value_error.missing)

[2023-09-27 00:03:41,694]::[uvicorn.error]::ERROR --> Application startup failed. Exiting.
Task was destroyed but it is pending!
task: <Task pending name='Task-3' coro=<FastAPIEventService.__dispatch_from_queue() running at E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\app\api\events.py:48> wait_for=<Future pending cb=[Task.task_wakeup()]>>
Desired action:
````





or Re-run the configure script to fix a broken install or to complete a major upgrade   ( choosing 7 )





error:

````
Desired action:
1. Generate images with the browser-based interface
2. Explore InvokeAI nodes using a command-line interface
3. Run textual inversion training
4. Merge models (diffusers type only)
5. Download and install models
6. Change InvokeAI startup options
7. Re-run the configure script to fix a broken install or to complete a major upgrade
8. Open the developer console
9. Update InvokeAI
10. Run the InvokeAI image database maintenance script
11. Command-line help
Q - Quit
Please enter 1-11, Q: [1] 7
Running invokeai-configure...
[2023-09-27 00:10:31,121]::[InvokeAI]::INFO --> Initializing InvokeAI runtime directory
[2023-09-27 00:10:31,141]::[InvokeAI]::INFO --> Scanning E:\DIFFUSION\InvokeAI\models for new models
Traceback (most recent call last):
  File "C:\Program Files\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "E:\DIFFUSION\InvokeAI\.venv\Scripts\invokeai-configure.exe\__main__.py", line 7, in <module>
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\install\invokeai_configure.py", line 910, in main
    models_to_download = default_user_selections(opt)
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\install\invokeai_configure.py", line 661, in default_user_selections
    installer = ModelInstall(config)
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\install\model_install_backend.py", line 108, in __init__
    self.mgr = model_manager or ModelManager(config.model_conf_path)
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\model_manager.py", line 354, in __init__
    self._read_models(config)
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\model_manager.py", line 377, in _read_models
    self.scan_models_directory()
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\model_manager.py", line 997, in scan_models_directory
    model_config: ModelConfigBase = model_class.probe_config(
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\models\base.py", line 206, in probe_config
    return cls.create_config(
  File "E:\DIFFUSION\InvokeAI\.venv\lib\site-packages\invokeai\backend\model_management\models\base.py", line 202, in create_config
    return configs[kwargs["model_format"]](**kwargs)
  File "pydantic\main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for CheckpointConfig
config
  field required (type=value_error.missing)
Desired action:
1. Generate images with the browser-based interface
2. Explore InvokeAI nodes using a command-line interface
3. Run textual inversion training
4. Merge models (diffusers type only)
5. Download and install models
6. Change InvokeAI startup options
7. Re-run the configure script to fix a broken install or to complete a major upgrade
8. Open the developer console
9. Update InvokeAI
10. Run the InvokeAI image database maintenance script
11. Command-line help
Q - Quit
Please enter 1-11, Q: [1]
````


I just stoop working during using application and crashing now ....

### Screenshots

_No response_

### Additional context

_No response_

### Contact Details

_No response_

**Comment by Millu** [comment]
Did you add a new model before this error occurred?

**Comment by mirek190** [comment]
Yes.

When application was working I added one sdxl lora  to designated folder.

But I was still able generating pictures ( I wasn't change models )  for about few minutes .
Then stopped generating suddenly so I closed Invoke completely and tried to run again ...but never run again  - just giving errors like above .

I was thinking maybe that lora what I added prevents to run Invoke so I deleted that lora but still not working and giving the same errors .

**Comment by Millu** [comment]
Does this error occur with a different base model as well?

**Comment by mirek190** [comment]
Don't know.
I saw that error first time in my life and posted here after few attempts trying to fix but failed .

Don't have time to download everything again and setup right now.

**Comment by mirek190** [comment]
> Does this error occur with a different base model as well?

What kind of diffeent mode?  I can not run Invoke at all right now ...even when I removed SDXL lora. 
I tried that sdxl lore before everything broke  -  face_xl_v0_1.safetensors

That lora was working - I generated few images and suddenly generation stopped working  I closed everything ... tried run again ..not working  ( above error )  , removed that lora still the same errors.
